src/components/power/Charging.js

```diff
@@ -183,7 +183,24 @@ class Charging extends Component {
                                 </div>
                             )
                         }, this)
-                    }
+                        }
+
+						{
+						(new Array(18)).map(function(_, cellIndex) {
+							return (
+								<div className="row" key={"cell" + cellIndex}>
+									<div>
+										<label>Module {cellIndex + 1} Volts</label>
+										<GenericParameterLabel
+											StreamingPageManager={_this.state.streamManager}
+											parameter="Power A BMS {index + 1} Module Voltage"/>
+									</div>
+									<button type="button" className="btn btn-success" onClick={this.startDischargeB.bind(this, {cellIndex: cellIndex})}  style={{margin:10}}>Start Discharging</button>
+									<button type="button" className="btn btn-success" onClick={this.stopDischargeB.bind(this, {cellIndex: cellIndex})}  style={{margin:10}}>Stop Discharging</button><br />
+								</div>
```

Comment:
None of these <divs> are showing up on the page, and I'm not sure why.  I'll try to get help from someone who knows JSX.